### PR TITLE
minor updates to getting started, wikis, and tables articles

### PR DESCRIPTION
**Pull request by ychae**

### CODE REVIEW

File: articles/tables.md [hunk 1]
```diff
@@ -95,9 +95,7 @@ For the following code examples, we'll be using the [Wondrous Research Project (
 import synapseclient
 syn = synapseclient.login()
 
-# Pandas (For Python only): Pandas is a popular library for working with tabular data. 
-# If you have Pandas installed, the goal is that Synapse Tables will play nice with it.
-# Convert a .csv to a Pandas DataFrame
+# Convert a .csv to a Pandas DataFrame (this requires that you have Pandas downloaded)
```
Comment (kdaily) [inline articles/tables.md]: pandas installed...

File: articles/tables.md [hunk 2]
```diff
@@ -164,8 +162,8 @@ If you upload a file, the Web interface will automatically detect the table sche
 The Table() function takes two arguments, a schema object and data in some form, which can be:
 
 - a path to a CSV file
-- a [Pandas](http://pandas.pydata.org/){:target="_blank"} [DataFrame](http://pandas.pydata.org/pandas-docs/stable/api.html#dataframe){:target="_blank"} (This is a Python-only feature)
-- a `RowSet` object
+- a dataframe
```
Comment (kdaily) [inline articles/tables.md]: No Pandas specific reference now?

File: articles/tables.md [hunk 3]
```diff
@@ -164,8 +162,8 @@ If you upload a file, the Web interface will automatically detect the table sche
...
+- a [`RowSet` object](http://docs.synapse.org/rest/org/sagebionetworks/repo/model/table/RowSet.html){:target="_blank"}
 - a list of lists where each of the inner lists is a row
```
Comment (kdaily) [inline articles/tables.md]: not sure this is accurate (though I see you got it from the Synapse Python docstring)

File: articles/tables.md [hunk 4]
```diff
@@ -199,7 +197,7 @@ Upload the table into Synapse by clicking the **Create** button.
 <br>
 
 **4. Query the table:**
-Python returns an iterator and R returns a data frame. To return a data frame in Python using Pandas `results.asDataFrame()`.
+Python returns an iterator and R returns a data frame. To return a data frame in Python use Pandas `results.asDataFrame()`.
```
Comment (kdaily) [inline articles/tables.md]: Add 'By default,'

File: articles/tables.md [hunk 5]
```diff
@@ -230,12 +228,7 @@ Tables can be queried by using the Query bar above each table.
 
 Once the schema is settled, changes can be made by adding, appending, and deleting.
 
-In Python, updating requires an *etag*, which identifies the most recent change set plus row IDs and version numbers for 
-each row to be modified. We get those by querying before updating. Minimizing changesets to contain only rows that actually 
-change will make processing faster.
-
-The etag is used by the server to prevent concurrent users from making conflicting changes, a technique called optimistic concurrency. 
-In case of a conflict, your update may be rejected. You then have to do another query and try your update again.
+When updating, begin by querying the table to ensure you have the latest schema.
```
Comment (kdaily) [inline articles/tables.md]: The latest schema, or (and?) latest table values?

File: articles/tables.md [hunk 6]
```diff
@@ -246,8 +239,8 @@ In case of a conflict, your update may be rejected. You then have to do another
 # Change the album value 'Vol. 2' to 'Volume 2' 
 results = syn.tableQuery("select * from %s where album='Vol. 2'" %table.schema.id)
 df = results.asDataFrame()
-df['album'] = ['Volume 2']
-syn.store(Table(schema, df, etag=results.etag))
+df['album'] = 'Volume 2'
+syn.store(Table(results, df))
```
Comment (kdaily) [inline articles/tables.md]: Confusing naming of `results`. Also, why do you need to create a new table to update?

File: articles/wikis.md [hunk 7]
```diff
@@ -74,6 +74,17 @@ Select **Tools** then select **Edit Project/Folder/File wiki**
 ### Markdown language
 The layout and text of a `Wiki` can be customized using Wiki markdown language. A Formatting Guide is available within the Wiki editing window. For additional markdown functions see [Markdown in Synapse](https://www.synapse.org/#!Wiki:syn2467792/ENTITY/64247){:target="_blank"}. Useful markdown shortcuts are available as separate tools, including: heading, bold, italic, strike-through, code block, sub and superscript. 
 
+### Tables in markdown
```
Comment (kdaily) [inline articles/wikis.md]: Tables in Markdown

File: articles/wikis.md [hunk 8]
```diff
@@ -74,6 +74,17 @@ Select **Tools** then select **Edit Project/Folder/File wiki**
...
+
+{% include tip.html content="Add this markdown below the header to enable sort on the header columns." %}
```
Comment (kdaily) [inline articles/wikis.md]: "Add this Markdown..."

File: articles/tables.md (hunk 2 above)
Comment (ychae) [inline articles/tables.md]: It seems more universal to say dataframe rather than a Pandas dataframe

Comment (kdaily) [inline articles/tables.md]: This is a Python specific section, yes? Python doesn't have data frames (other than in Pandas).

**Pull request merged**